### PR TITLE
Change empty links to buttons

### PR DESCRIPTION
**Pull request by imjuniper**
Changes all links that have an href set to `#` to buttons instead.

### CODE REVIEW

**Comment by imjuniper** [comment]
Actually I think the JS bug got fixed before, but still.

**Comment by imjuniper** [comment]
Ah damn... Seems that type="button" is necessary so that it doesn't submit forms. How weird and stupid.

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
Changes made here look fine. 

Have noticed some other minor bugs / strange effects with the new javascript see Discord

File: web/css/src/themes/default.css [hunk 1]
```diff
@@ -239,14 +240,16 @@
...
 	display: block;
 	background-color: transparent;
-	border: 0;
-	color: inherit;
-	width: 100%;
 	text-align: center;
+	border: 0;
 	padding: 10px 15px;
+	width: 100%;
+	background: none;
```
Comment (AlecRust) [inline web/css/src/themes/default.css:247]: Doesn't look like any need to set shorthand here, `background-color` should do?

Comment (imjuniper) [reply web/css/src/themes/default.css:247]: indeed, I'll change that. I have a bad habit of using shorthands unnecessarily that I'm trying to change haha

**Pull request merged**